docker/Dockerfile.contrib

```diff
@@ -1,25 +1,33 @@
-FROM node:latest AS build
+FROM node:erbium-stretch AS build
 USER root
+RUN apt-get update && apt-get install -y jq
 RUN mkdir -p /usr/local/lib/node_modules && chown node /usr/local/lib/node_modules
 USER node
 
 COPY --chown=node node-zwave-js /home/node/node-zwave-js
 COPY --chown=node zwavejs2mqtt /home/node/zwavejs2mqtt
 
 WORKDIR /home/node/node-zwave-js
-RUN npm install
-RUN npm run build
-RUN npm link
+RUN rm -f package-lock.json
+RUN yarn install
+RUN yarn run build
+RUN yarn install --production --frozen-lockfile
+RUN npx lerna exec -- yarn link
```

Comment:
Interesting combination of commands ;)
![grafik](https://user-images.githubusercontent.com/17641229/100594512-11655b00-32fa-11eb-923b-35eb30bf098f.png)

Reply:
![magic](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)